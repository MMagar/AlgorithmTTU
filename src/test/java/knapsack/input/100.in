2500
160 88
153 28
176 50
161 28
174 82
175 82
171 56
137 22
145 21
187 93
138 36
195 96
199 54
168 83
103 19
112 20
163 95
156 50
103 25
179 79
136 29
170 32
139 39
175 95
173 39
166 69
196 73
187 98
197 85
180 57
197 92
147 39
180 81
173 75
154 80
145 39
111 16
142 22
187 81
157 28
164 48
186 65
141 47
180 76
163 45
112 22
136 38
194 66
177 63
174 94
173 81
194 81
130 32
110 22
137 48
123 17
185 58
156 44
185 73
158 66
168 81
134 40
101 19
163 65
171 75
162 54
160 34
161 38
197 65
150 48
131 21
130 35
107 18
166 56
164 27
145 36
178 37
178 96
171 51
182 93
182 62
169 26
104 24
175 32
166 79
160 72
163 59
149 50
138 15
144 37
163 97
124 20
151 61
162 60
148 20
169 66
172 43
177 48
181 69
153 69
