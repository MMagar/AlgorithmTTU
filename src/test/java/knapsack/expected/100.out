9783 2500
138 15
148 20
123 17
111 16
145 21
169 26
142 22
131 21
137 22
124 20
164 27
107 18
161 28
157 28
112 20
175 32
153 28
103 19
101 19
170 32
112 22
110 22
178 37
160 34
136 29
173 39
104 24
161 38
103 25
130 32
145 36
172 43
144 37
138 36
147 39
145 39
130 35
177 48
199 54
163 45
136 38
139 39
156 44
176 50
164 48
171 51
134 40
185 58
180 57
150 48
156 50
171 56
197 65
141 47
162 54
149 50
166 56
194 66
182 62
186 65
137 48
177 63
163 59
162 60
151 61
